assets/templates/partials/calendar/items/subscription-links.tmpl

```diff
@@ -32,6 +32,7 @@
       <span class="ons-list__prefix" aria-hidden="true">
         <img
           class="ons-svg-icon ons-u-d-b"
+          style="max-width: none !important;"
```

Comment:
inline style? 🤨

Reply:
I know 😢

Basically without it the calendar icon behaves a bit weird when the window gets resized i.e. the smaller the window the tinier it gets until it disappears so I think this css is needed. 

I kinda feel it is a good candidate tho for inline styling as taboo as it is, as I don't think this styling should be used anywhere else so don't wish to make it a class.


Glad to hear your thoughts tho.

Reply:
In case anyone else stumbles across this...the merge in from develop seems to have resolved this 🤷‍♂️ so the inline style has been removed ☮️